### ISSUE TITLE
自己加装OKS loss但效果不好

### ISSUE THREAD

**Issue by yqsong1** [xialuxi/yolov5-car-plate#20]
大佬好，
想问下怎么样才能自己加装OKS loss呢，我拿你的代码改加了针对landmark的OKS loss（plate_loss.py, line 280左右）：

```
d = (plandmarks[:,0::2]-tlandmarks[i][:,0::2])**2 + (plandmarks[:,1::2]-tlandmarks[i][:,1::2])**2
s = torch.prod(tbox[i][:,-2:], dim=1, keepdim=True)
lmark += (1 -(torch.exp(-d/((4*sigmas**2)*s+1e+3)))).mean()
```

但是检测效果很差，想问问到底哪里有问题呢

**Comment by xialuxi** [comment]
如想要换成OKS loss， 需要把点的偏移换算成图像坐标再计算。
